packages/mongo-menu/src/MongoMenu.spec.tsx

```diff
@@ -57,4 +65,24 @@ describe('packages/MongoMenu', () => {
 
     expect(onLogout).toHaveBeenCalledTimes(1);
   });
+
+  test('renders the account link as a disabled button when set to the empty string', () => {
+    renderedComponent.rerender(
+      <MongoMenu
+        user={{ name: 'Leafy', email: 'leafy@mongodb.com' }}
+        activeProduct={Product.Atlas}
+        onLogout={onLogout}
+        onProductChange={onProductChange}
+        accountURL={''}
+      />,
+    );
+
+    const accountItem = getByText('MongoDB Account').parentElement;
+    const accountButton = accountItem as HTMLButtonElement;
```

Comment:
You're typing accountItem twice here. We should be able to rely on:
```ts
    const accountItem: HTMLButtonElement = getByText('MongoDB Account').parentElement;
```

Then we can remove the definition of `accountButton` altogether, and the inline typing within the assertion below.

Reply:
Yeah this is no longer relevant anyway, since this will be an `HTMLAnchorElement` as per your feedback on the component itself. However that does mean that unfortunately we can't test for `accountItem.disabled` since `disabled` is not a property on `HTMLAnchorElement`, without doing some more digging into typing. Aria props, however, will still be testable.